tracer/src/Datadog.Tracer.Native/iast/method_analyzers.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+#include "method_analyzer.h"
+#include "../../../../shared/src/native-src/pal.h"
+#include <atomic>
+#include <map>
+#include <set>
+
+#include "hardcoded_secrets_method_analyzer.h"
+
+using namespace shared;
+
+namespace iast
+{
+    class MethodInfo;
+
+    class MethodAnalyzers
+    {
+    public:
+        inline static std::vector<MethodAnalyzer*> InitAnalyzers()
+        {
+            std::vector<MethodAnalyzer*> res;
+            res.push_back(new HardcodedSecretsMethodAnalyzer());
```

Comment:
why not using `smart pointers` instead raw pointers ?

Reply:
The force of habit of a grumpy old man XD

Reply:
🤣 No worries